### PR TITLE
SLES-15-010357 rule

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _SLES-15-010357 rule_

#### Rationale:

- _Review required_

- Fixes # _Issue number here (e.g. #26) or remove this line if no issue exists._

### CODE REVIEW

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_system_commands_dirs/oval/shared.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  </definition>
+
+  <unix:file_test check="all" check_existence="none_exist" comment="system commands go-w" id="test_perms_system_commands_dirs" version="1">
+    <unix:object object_ref="object_file_permissions_system_commands_files" />
+  </unix:file_test>
+
+  <unix:file_object comment="binary files" id="object_file_permissions_system_commands_files" version="1">
+    <!-- Check that binary files under /bin, /sbin, /usr/bin, /usr/sbin, /usr/local/bin,
+         and /usr/local/sbin directories have safe permissions (go-w) -->
+    <unix:path operation="pattern match">^\/[s?]bin|^\/usr\/[s?]bin|^\/usr\/local\/[s?]bin</unix:path>
```
[the remark this answers is not in the document]
Comment (rumch-se) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_system_commands_dirs/oval/shared.xml]: @jan-cerny - I fixed this

**Comment by vojtapolasek** [comment]
@openscap-ci ok to test

**Comment by vojtapolasek** [comment]
@openscap-ci test this please

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_system_commands_dirs/rule.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+documentation_complete: true
+
+prodtype: sle15
+
+title: 'Verify that system commands are protected from unathorized access'
```
Comment (vojtapolasek) [inline linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_system_commands_dirs/rule.yml:5]:
```suggestion
title: 'Verify that system commands are protected from unauthorized access'
```

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_system_commands_dirs/ansible/shared.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = medium
+# disruption = medium
+
+
+- name: "Retrive the system command files and protect them from unautorized access"
```
Comment (vojtapolasek) [inline linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_system_commands_dirs/ansible/shared.yml:8]:
```suggestion
- name: "Retrieve the system command files and protect them from unautorized access"
```

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello, thank you for the rule and tests, it looks really good. Please could you add just one more test which uses symlink?
Thank you.

**Comment by ggbecker** [comment]
@openscap-ci test this please

**Review by vojtapolasek — APPROVED** [review]

**Pull request merged**